com.kms.cura.entity/src/com/kms/cura/entity/MessageEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.kms.cura.entity;
+
+import java.sql.Timestamp;
+import com.kms.cura.entity.user.DoctorUserEntity;
+import com.kms.cura.entity.user.UserEntity;
+
+public class MessageEntity extends Entity {
+	private UserEntity sender, receiver;
+	private Timestamp timeSent;
```

Comment:
Should not create field for time with type java.sql.Timestamp. Use Date instead.